tortoise/sim/layer.go

```diff
@@ -108,8 +108,7 @@ func (g *Generator) Next(opts ...NextOpt) types.LayerID {
 
 func (g *Generator) genBeacon() {
 	eid := g.nextLayer.Sub(1).GetEpoch()
-	beacon := make([]byte, 32)
-	g.rng.Read(beacon)
+	beacon := types.RandomBeacon()
```

Comment:
btw please use internal g.rng here. the idea is that if seed is the same - all generated objects are the same

Reply:
thanks for the explanation. make sense. changed as advised